### ISSUE TITLE
Please disable the blur effect on press middle mouse button 

### ISSUE THREAD

**Issue by Gummibaer** [linuxserver/ifai#1044]
Should be fine in 2.5.3. The blur effect is mainly for keyboard navigation to make it clear which item is highlighted (when using tab to cycle through tiles).

_Originally posted by @KodeStar in https://github.com/linuxserver/Heimdall/issues/1042#issuecomment-1328227249_
      

Please disable the blur effect on press middle mouse button

**Comment by KodeStar** [comment]
Will be fixed in 2.5.4 (code is in at the tip of development, just waiting for a release).
